playwright.config.js

```diff
@@ -51,6 +51,32 @@ export default defineConfig({
         ...devices['Desktop Firefox'],
         baseURL: process.env.BASE_URL
       }
+    },
+    {
+      /**
+       * Test the site with service workers disabled. You need to `import {testNoServiceWorker as test, expect} from './fixtures/config-test-fixtures.js'` to use this project.
+       * Anything needing a service worker will be skipped when this project is ran.
+       */
+      name: 'no-service-worker',
+      testMatch: /test-e2e\/no-service-worker\.test\.ts/,
+      use: {
+        ...devices['Desktop Firefox'],
```

Comment:
Note that this doesn't test no service worker functionality in chrome, but we shouldn't need to...?